docs/ui.md

```diff
@@ -27,6 +27,79 @@ These elements are defined in the manifest.xml file:
 </connector-plugin>
 ```
 
+## Vendor Attributes (Custom Fields)
+Vendors can add customized attributes to their connector plugin by using the vendor attributes.
+
+These fields have a custom label and can be used for attributes in the connection strings that are not available in the attribute list. You can currently add 3 custom fields in your connector plugin.
+
+To add a custom vendor-attribute, you will need to modify your connection-dialog.tcd, connectionResolver.tdr and connectionBuilder.js.
+**For JDBC based plugins, you will need to modify connectionProperties.js instead of connectionBuilder.js**
+
+connection-resolver.tdr
+
+```
```

Comment:
@d45 or @kpwilson are we able to format code blocks in markdown by adding the name of the language in the code block after the three backticks? e.g. ` ```xml `?

Reply:
It seems it does. Tried doing it locally. 
![xmlPlatform](https://user-images.githubusercontent.com/14967333/67123070-1de52d80-f1a4-11e9-8e5c-451fd9017242.png)

Reply:
Awesome! I think it'd be worth adding those to the code blocks. Makes everything so much easier to read.